### PR TITLE
Update to and at; selections for grow; add seam

### PR DESCRIPTION
**Pull request by pentacular**

### CODE REVIEW

File: algorithm/cgal/cgal.cc [hunk 1]
```diff
@@ -5243,6 +5290,32 @@ int Outline(Geometry* geometry) {
   return STATUS_OK;
 }
 
+int Seam(Geometry* geometry, size_t count) {
```
Comment (BarbourSmith) [inline algorithm/cgal/cgal.cc:5293]: Is Seam already working? I thought the plan was to add that at some point in the future?

File: nb/regression/shapes/shapes.md [hunk 2]
```diff
@@ -413,10 +405,71 @@ Orb(10)
...
+![Image](shapes.md.39.png)
+
+```JavaScript
+blue.fitTo(red.to(blue)).view();
+```
+
+![Image](shapes.md.40.png)
+
+```JavaScript
+red.at(blue, cut(Box([10, 20], 20))).view();
```
Comment (BarbourSmith) [inline nb/regression/shapes/shapes.md:466]: Nice 🙂

**Review by BarbourSmith — APPROVED** [review]
Other than the tests, looks good to me 👍 👍

File: algorithm/cgal/cgal.cc (hunk 1 above)
Comment (pentacular) [reply algorithm/cgal/cgal.cc:5293]: Yeah, I wanted to see how it worked -- I think it'll require a bit of refinement, but it was easy enough to do.

**Pull request merged**